omnisafe/algorithms/on_policy/ppo_lag.py

```diff
@@ -12,36 +12,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+"""Implementation of the PPO-Lag algorithm."""
 
 import torch
 
-from omnisafe.algos import registry
-from omnisafe.algos.common.lagrange import Lagrange
-from omnisafe.algos.on_policy.policy_gradient import PolicyGradient
+from omnisafe.algorithms import registry
+from omnisafe.algorithms.on_policy.policy_gradient import PolicyGradient
+from omnisafe.common.lagrange import Lagrange
 
 
 @registry.register
 class PPOLag(PolicyGradient, Lagrange):
-    def __init__(self, algo='ppo-lag', clip=0.2, **cfgs):
-        PolicyGradient.__init__(self, algo=algo, **cfgs)
-        Lagrange.__init__(self, **self.cfgs['lagrange_cfgs'])
+    """specific functions"""
```

Comment:
Invalid docstring. This may be a comment.

Reply:
All algorithms will be change to
```
    """The Responsive Safety in Reinforcement Learning by PID Lagrangian Methods.

    References:
        Paper Name: Responsive Safety in Reinforcement Learning by PID Lagrangian Methods.
        Paper author: Joshua Achiam, David Held, Aviv Tamar, Pieter Abbeel.
        Paper URL: https://arxiv.org/abs/1705.10528

    """
```